### PR TITLE
Update free-courses-bn.md

### PR DESCRIPTION
**Pull request by jonyahmed19**
Added Mern stack bangla playlist

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes
It seems the linter is not enforcing alphabetical order in Bangla?

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -122,6 +122,7 @@
 
 #### Node.js
 
+* [Full MERN Stack Course](https://www.youtube.com/watch?v=ewBBT6Iph0M&list=PL_XxuZqN0xVD0op-QDEgyXFA4fRPChvkl) - Stack Learner
 * [Dive Into NodeJS](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDHFj-ecFSU0SU-B0TuJRk9) - Stack Learner
```
Comment (davorpa) [inline courses/free-courses-bn.md:126]: Alphabetize, update resource title and use playlist url

```suggestion
* [Dive Into NodeJS](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDHFj-ecFSU0SU-B0TuJRk9) - Stack Learner
* [Full-stack Army - Complete MERN Stack Journey in Bangla](https://www.youtube.com/playlist?list=PL_XxuZqN0xVD0op-QDEgyXFA4fRPChvkl) - Stack Learner
```

**Review by davorpa — CHANGES_REQUESTED** [review]
It's a good moment to apply some changes. When done, request for a review again using GitHub :arrows_clockwise: button. Meanwhile, draft status can be used to work on it if you want.

**Comment by davorpa** [comment]
> For youtube, we prefer "playlist" links and the name of the producer/creator. [https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md?rgh-link-date=2022-06-30T01%3A56%3A16Z#platforms-and-access-notes) It seems the linter is not enforcing alphabetical order in Bangla?

It seems :worried:.

**English should be used if some language is not registered in linter.**

But Bangla is one of these languages added at Hacktoberfest 2021 pending to add to linter (see backlog #5762)

**Review by jonyahmed19 — COMMENTED** [review]
I have changed to alphabetical order

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**